app-react/src/main/kotlin/com/ustadmobile/view/ContentEntryDetailOverviewScreen.kt

```diff
@@ -0,0 +1,483 @@
+package com.ustadmobile.view
```

Comment:
Please add this to useustadscreens so that when running it will appear in the list.